1.purpose_and_goals.md

```diff
@@ -6,12 +6,15 @@ For example, a contemporary cloud application may be composed of dozens of micro
 
 This specification provides a description of such applications, where the description is aimed at separating the concerns of developers from the concerns of operators. Furthermore, it suggests patterns and processes for managing such applications. The specification describes a model for cloud native (i.e. highly distributed) cloud applications, encompassing public cloud technologies, on-prem solutions, and IoT/edge technologies. By specifying a common model, this specification provides a foundation for multiple implementations of a shared description of cloud native applications.
 
-This open specification defines:
+This open specification defines the following model:
 
-- __Component schematics__, where developers declare the operational characteristics of the code they deliver _in infrastructure neutral terms_.
-- __Application scopes__, a way to group components into loosely coupled applications with common characteristics.
+![Conceptual Model Diagram](./assets/conceptual-model-diagram.png)
+
+- __Component__, where developers declare the operational characteristics of the code they deliver _in infrastructure neutral terms_.
+- __Scopes__, a way to loosely couple components into groups with common characteristics.
+- __Applications__, where developers group components together into a single, deployable unit and specifies cross-component info, such as health scopes.
```

Comment:
This point of _Applications_ concept does not exist yet.
It is related to https://github.com/oam-dev/spec/pull/148, but for now we might want to hold or modify this PR.